topics/single-cell/tutorials/scrna-case_alevin-combine-datasets/tutorial.md

```diff
@@ -168,9 +168,9 @@ You can access the data for this tutorial in multiple ways:
 > 1. {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.10.3+galaxy0) %} with the following parameters:
 >    - {% icon param-file %} *"Annotated data matrix"*: `N701-400k`
 >    - *"Function to manipulate the object"*: `Concatenate along the observations axis`
->    - {% icon param-file %} *"Annotated data matrix to add"*: `Select all the other matrix files from bottom to top, N707 to N702`
+>    - {% icon param-file %} *"Annotated data matrix to add"*: `Select all the other matrix files from bottom to top, N702 to N707` (if you imported files from Zenodo, your history may not be in this order, but you can still select them in order from N702 to N707 using the file names)
 >
->    > <warning-title>N707 to N702!</warning-title>
+>    > <warning-title>N701 to N701!</warning-title>
```

Comment:
those are now the same ...?

Reply:
I've changed this now as I think both versions weren't really making the main point to not add N701!